web/src/main/webapp/WEB-INF/pages/candidate_create.jsp

```diff
@@ -109,11 +109,9 @@
                                 <input type="text" class="input form-control" name="email" id="email" data-bind="valueWithInit: 'email'"
                                        value="${candidate.email}"
                                        placeholder="${i18nemail}"
-                                       data-error="<spring:message
-                                    code="candidate.error.email.empty"/>"
+                                       <spring:message code="candidate.error.email.empty" var="messValue"/>data-error="${messValue}"
                                        data-pattern="[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}"
-                                       data-pattern-error="<spring:message
-                                    code="candidate.error.email.incorrect"/>"
+                                      <spring:message code="candidate.error.email.incorrect" var="codeValue"/> data-pattern-error="${codeValue}"
```

Comment:
Use better naming. The `codeValue` does not tell that it contains the "incorrect email" error message.